macros/edr/tests/test_utils/validate_test_parameters.sql

```diff
@@ -11,4 +11,15 @@
         {% do exceptions.raise_compiler_error('Seasonality is supported only with time_bucket 1 day, got period: ' ~ time_bucket.period ~ ' and count: ' ~ time_bucket.count ~ ' instead') %}
       {% endif %}
     {% endif %}
+{% endmacro %}
+
+{% macro validate_directional_parameter(anomaly_direction) %}
+    {% if anomaly_direction %}
+      {% set direction_case_insensitive = anomaly_direction | lower %}
+      {% if direction_case_insensitive != 'drop' and direction_case_insensitive != 'spike' and direction_case_insensitive != 'both' %}
```

Comment:
Not critical but for next time:
`direction_case_insensitive not in ['drop','spike',both']`